src/RenderWebGL.js

```diff
@@ -1495,6 +1518,51 @@ class RenderWebGL extends EventEmitter {
         // Simplify boundary points using convex hull.
         return hull(boundaryPoints, Infinity);
     }
+
+    static sampleColor4b (vec, drawable, dst) {
+        const localPosition = Drawable.getLocalPosition(drawable, vec);
```

Comment:
I wonder if there would be an improvement by listing a separate copy of `getLocalPosition` in this file. I think this may mean this method would be better suited on `Drawable` than `RenderWebGL`.